src/main/java/uk/gov/companieshouse/web/pps/models/EnterDetails.java

```diff
@@ -32,16 +17,14 @@ public EnterDetails(PenaltyReference penaltyReference) {
     @Pattern(regexp = "^([a-zA-Z0-9]{8}|\\d{1,8})$", message = "{enterDetails.companyNumber.wrongLength}")
```

Comment:
Can this be changed to "{enterDetails.companyNumber.invalid}" as it's not just the length that it is checking, it is also the format.

Reply:
See other comment, invalid / notValid could work, but if other references use wrong we can either update those or leave everything as is.

Reply:
Fixed in commit [54fb8a1](https://github.com/companieshouse/penalty-payment-web/pull/180/commits/54fb8a152d4f8c1ee331f247866d23cabdf57278). I've updated the message key names to be `notEmpty` and `notValid` in `ValidationMessages.properties` following the comments.